pkgs/development/libraries/qt-5/modules/qtwayland.nix

```
@@ -4,7 +4,7 @@ qtModule {
   pname = "qtwayland";
   qtInputs = [ qtbase qtquickcontrols ];
   buildInputs = [ wayland ];
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config wayland ];
```

```suggestion
  nativeBuildInputs = [ pkg-config wayland-scanner ];
```


It should be possible to use `wayland-scanner,` which is an alias of `wayland.bin`

wayland-scanner will split from wayland in future：https://github.com/NixOS/nixpkgs/pull/214906